examples/generated/test/react-dom/shadows.js

```diff
@@ -0,0 +1,7 @@
+import colors from "./colors"
+
+export default {
+  elevation1: { filter: "drop-shadow(0px 1px 2px rgba(0,0,0,0.5));" },
+  elevation2: { filter: "drop-shadow(0px 2px 4px rgba(0,0,0,0.5));" },
+  elevation3: { filter: "drop-shadow(0px 3px 9px " + colors.grey900 + ");" }
```

Comment:
This object format seems like the best approach, since at some point we could add a flag to use box-shadow or drop-shadow, and we wouldn't have to change anything elsewhere.

The other option would be to use just the string:

`elevation1: "drop-shadow(0px 1px 2px rgba(0,0,0,0.5));"`

And then in the JSX, `filter: shadows.elevation1`. That might sometimes result in one less object spread, but other than that I don't see any advantage.

Are there any other tradeoffs? How did you choose this format?

Reply:
I went this direction just in case we had to augment the css with more than just filter to reproduce the right shadow. It felt the most flexible, but I'm open to changing it if you'd like

Reply:
I think this is good. Let's go with this, and we can always revisit if a new use case or something comes up.